### PR TITLE
mon: '* list' -> '* ls'

### PR DESCRIPTION
**Pull request by liewegas**
There are just few legacy commands not following current style.

### CODE REVIEW

**Comment by theanalyst** [comment]
PendingReleaseNotes entry

**Comment by liewegas** [comment]
updated release-notes.rst directly

File: qa/workunits/cephtool/test.sh [hunk 1]
```diff
@@ -537,6 +537,7 @@ function test_auth()
   ceph auth add client.xx -i client.xx.keyring
   rm -f client.xx.keyring
   ceph auth list | grep client.xx
```
Comment (xiexingguo) [inline qa/workunits/cephtool/test.sh]: And drop this line?

Comment (xiexingguo) [inline qa/workunits/cephtool/test.sh]: Well, I am little confused. Do you mean we should test both formats?

Comment (tchaikov) [inline qa/workunits/cephtool/test.sh]: we should before "auth list" is dropped.

File: qa/workunits/mon/crush_ops.sh [hunk 2]
```diff
@@ -14,6 +14,7 @@ ceph osd crush dump
 ceph osd crush rule dump
 ceph osd crush rule ls
 ceph osd crush rule list
+ceph osd crush rule ls
```
Comment (tchaikov) [inline qa/workunits/mon/crush_ops.sh]: please drop this duplicated test, see https://github.com/ceph/ceph/pull/16423/files#diff-bfe2ac67a1e18313cc7532a6a24dbad6R15

File: qa/workunits/cephtool/test.sh [hunk 3]
```diff
@@ -575,9 +576,9 @@ function test_auth()
   check_response "auid = $auid"
   ceph --format json-pretty auth get client.TEST > $TMPFILE
   check_response '"auid": '$auid
-  ceph auth list > $TMPFILE
+  ceph auth ls > $TMPFILE
```
Comment (xiexingguo) [inline qa/workunits/cephtool/test.sh]: @tchaikov Thanks, kefu.  I get confused because Sage directly updates the old format here and below.

**Comment by liewegas** [comment]
I left that in to verify the old command is still present (for a 
transition period); eventually it will be removed.

**Pull request merged**